lib/shared/drive-constraints.js

```diff
@@ -276,6 +276,29 @@ exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE)
 }
 
+/**
+ * @summary 64GB
+ * @private
+ * @constant
+ */
+exports.LARGE_DRIVE_SIZE = 64e9
```

Comment:
Is the `@private` label correct, given that it's included in `exports`? If it's private, does that mean it shouldn't really be used in `tests/shared/drive-constraints.spec.js` ? \*shrug\*

Reply:
LOL, only just noticed that you made these changes at @jviotti 's request. Ignore my comment then :wink: